www/js/strings.json

```diff
@@ -0,0 +1,83 @@
+{
+  "moolMantra": {
+    "gurmukhi": "<> siq nwmu krqw purKu inrBau inrvYru Akwl mUriq AjUnI sYBM gur pRswid ]",
+    "english": "One Universal Creator God. The Name Is Truth. Creative Being Personified. No Fear. No Hatred. Image Of The Undying, Beyond Birth, Self-Existent. By Guru's Grace ~",
+    "punjabi": "ਅਕਾਲ ਪੁਰਖ ਇੱਕ ਹੈ, ਜਿਸ ਦਾ ਨਾਮ 'ਹੋਂਦ ਵਾਲਾ' ਹੈ ਜੋ ਸ੍ਰਿਸ਼ਟੀ ਦਾ ਰਚਨਹਾਰ ਹੈ, ਜੋ ਸਭ ਵਿਚ ਵਿਆਪਕ ਹੈ, ਭੈ ਤੋਂ ਰਹਿਤ ਹੈ, ਵੈਰ-ਰਹਿਤ ਹੈ, ਜਿਸ ਦਾ ਸਰੂਪ ਕਾਲ ਤੋਂ ਪਰੇ ਹੈ, (ਭਾਵ, ਜਿਸ ਦਾ ਸਰੀਰ ਨਾਸ-ਰਹਿਤ ਹੈ), ਜੋ ਜੂਨਾਂ ਵਿਚ ਨਹੀਂ ਆਉਂਦਾ, ਜਿਸ ਦਾ ਪ੍ਰਕਾਸ਼ ਆਪਣੇ ਆਪ ਤੋਂ ਹੋਇਆ ਹੈ ਅਤੇ ਜੋ ਸਤਿਗੁਰੂ ਦੀ ਕਿਰਪਾ ਨਾਲ ਮਿਲਦਾ ਹੈ।",
+    "translit": " ikOankaar sat naam karataa purakh nirabhau niravair akaal moorat ajoonee saibha(n) gur prasaadh ||"
+  },
+  "waheguru": "vwihgurU",
+  "dhanSlides": [
+    {
+      "gurmukhi": "DMn gurU nwnk dyv jI",
+      "english": "Dhan Guru Nanak Dev Ji",
```

Comment:
@saintsoldierx do we want to translate "Dhan" for the English or just leave it like this? 

"Blessed is Guru Nank Dev Ji" ?

Reply:
i think if we're going to do translations, 'dhan' should probs translate to 'great'

Reply:
+1 to "Great"

Reply:
should we add "Sahib"

Reply:
"Great" is fine. Please do add Sahib. Normally when sung, we sing "Dhan Guru Nanak Dev Sahib Jee, Dhan Guru Angad Dev Sahib Ji" etc..